default.py

```diff
@@ -143,108 +143,82 @@ def display_nfl_network_archive(self):
-                                    game_info = game['phase']
-                        else:
-                            game_info = 'FINAL'
-                        listitem.setProperty('game_info', game_info)
-                    else:
-                        if 'isLive' in game:
-                            game_info = '» Live «'
-
-                        try:
-                            if addon.getSetting('local_tz') == '0':  # don't localize
-                                game_datetime = datetime.datetime(*(time.strptime(game['gameDateTimeUtc'], date_time_format)[0:6]))
-                                game_info = game_datetime.strftime('%A, %b %d - %I:%M %p')
-                            else:
-                                from_zone = tz.tzutc()
-                                to_zone = tz.tzlocal()
-                                game_datetime = datetime.datetime(*(time.strptime(game['gameDateTimeUtc'], date_time_format)[0:6]))
-                                game_datetime = game_datetime.replace(tzinfo=from_zone)
-                                local_time = game_datetime.astimezone(to_zone)
-                                if addon.getSetting('local_tz') == '1':  # localize and use 12-hour clock
-                                    game_info = local_time.strftime('%A, %b %d - %I:%M %p')
-                                else:  # localize and use 24-hour clock
-                                    game_info = local_time.strftime('%A, %b %d - %H:%M')
-                        except:  # all else fails, just use their raw date value
-                            game_datetime = game['gameDateTimeUtc'].split('T')
-                            game_info = game_datetime[0] + '[CR]' + game_datetime[1].split('.')[0] + ' ET'
-                        listitem.setProperty('game_info', game_info)
-
-                    if weekSet == 'weekScheduledGames':
-                        isPlayable = 'false'
-                        isBlackedOut = 'false'
+        for game in games:
+            game_id = '{0}-{1}-{2}'.format(game['visitorNickName'].lower(), game['homeNickName'].lower(), str(game['gameId']))
+            game_name_shrt = '[B]%s[/B] at [B]%s[/B]' % (game['visitorNickName'], game['homeNickName'])
+            game_name_full = '[B]%s %s[/B] at [B]%s %s[/B]' % (game['visitorCityState'], game['visitorNickName'], game['homeCityState'], game['homeNickName'])
+            listitem = xbmcgui.ListItem(game_name_shrt, game_name_full)
+
+            listitem.setProperty('is_game', 'true')
+            listitem.setProperty('is_show', 'false')
+
+            if game['phase'] == 'FINAL' or game['phase'] == 'FINAL_OVERTIME':
+                # show game duration only if user wants to see it
+                if addon.getSetting('hide_game_length') == 'false':
+                    game_info = '%s [CR] Duration: %s' % (game['phase'], str(datetime.timedelta(seconds=int(float(game['video']['videoDuration'])))))
```

Comment:
Just noting, in case the issue comes back to bite us: there were a lot of oddities with datetime and Kodi on some platforms. We'll just keep this change in mind if we get bug reports and need to put the "try" statement back in.

---Alex